README.md

```diff
@@ -76,25 +76,22 @@ repo](https://github.com/rapidsai/notebooks-contrib).
 | | Uniform Manifold Approximation and Projection (UMAP) | |
 | | Random Projection | |
 | | t-Distributed Stochastic Neighbor Embedding (TSNE) | |
-| **Linear Models for Regression or Classification** | Linear Regression (OLS) | Multi-GPU available in conda CUDA 10 package |
-| | Linear Regression with Lasso or Ridge Regularization | |
+| **Linear Models for Regression or Classification** | Linear Regression (OLS) | Experimental multi-GPU available in conda CUDA 10 package |
+| | Linear Regression with Lasso or Ridge Regularization | Experimental multi-node multi-GPU via Dask|
 | | ElasticNet Regression | |
 | | Logistic Regression | |
 | | Stochastic Gradient Descent (SGD), Coordinate Descent (CD), and Quasi-Newton (QN) (including L-BFGS and OWL-QN) solvers for linear models  | |
 | **Nonlinear Models for Regression or Classification** | Random Forest (RF) Classification | Experimental multi-node multi-GPU via Dask |
 | | Random Forest (RF) Regression | Experimental multi-node multi-GPU via Dask |
-|  | K-Nearest Neighbors (KNN) Classification | Multi-node multi-GPU via Dask, available in version 0.12 branch and nightly conda packages. Uses [Faiss](https://github.com/facebookresearch/faiss) for Nearest Neighbors Query. |
-|  | K-Nearest Neighbors (KNN) Regression | Multi-node multi-GPU via Dask, available in version 0.12 branch and nightly conda packages. Uses [Faiss](https://github.com/facebookresearch/faiss) for Nearest Neighbors Query.  |
+|  | K-Nearest Neighbors (KNN) Classification | Multi-node multi-GPU via Dask, it uses [Faiss](https://github.com/facebookresearch/faiss) for Nearest Neighbors Query. |
```

Comment:
```suggestion
|  | K-Nearest Neighbors (KNN) Classification | Multi-node multi-GPU via Dask, uses [Faiss](https://github.com/facebookresearch/faiss) for Nearest Neighbors Query. |
```

better without the `it` pronoun

Reply:
The MNMG NearestNeighbors doesn't yet support Classification/Regresion. This will be coming in a future release (hopefully very soon). Currently, the KNeighborsClassifier and KNeighborsRegressor are single GPU